modules/backend/formwidgets/repeater/partials/_repeater_item.htm

```diff
@@ -11,7 +11,7 @@
             <div class="repeater-item-handle <?= $this->getId('items') ?>-handle">
                 <i class="icon-bars"></i>
             </div>
-        <?php endif; ?>
+        <?php endif ?>
```

Comment:
oy what's this then???

Reply:
yeah, the semicolon is required there!

Reply:
Even if it's not it's required for my sanity 😜 Same for the endif in the other file.